Assets/Scripts/Importing/ImageSequenceImporter/SimpleITK/SimpleITKDICOMImporter.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: c7176da21a65a4fbbab548f6227ab34a
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
What is this file being used for? It looks like something the ITK installation might have left behind so I'm asking to make sure there is no conflict in the future.

Reply:
.meta files are automatically created, one for every file and a folder in a Unity project. They store some essential info about the actual files they belong to. Unfortunately, we'll have to track them in git and cannot get rid of them.